### ISSUE TITLE
Improve frontend performance on diagnosis page

### ISSUE THREAD

**Issue by gambala** [Neodelf/lecter#49]
Tested this lib in one of my production ready project, and for some reason I see 1163 files on diagnosis page. Overall page performance (scrolling and interacting with file collapsing nodes) can be improved – by reducing amount of DOM nodes, CSS repaint hacks, removing heavy js events, etc

**Comment by gambala** [comment]
Feel free to add me as a contributor and assign to this issue – will fix it

**Comment by Neodelf** [comment]
I used ready-made templates. So yep, the frontend part required to be improved 😄 
> Feel free to add me as a contributor 
As I know anyone could be a contributor. Please feel free to write me details

**Comment by gambala** [comment]
Yeah I mean giving me a permission to push commits to master/main branch directly, without forking and PRs. The same process is in https://github.com/estum/russian_inflect for example

**Comment by Neodelf** [comment]
I am not 100% sure about that approach. Yes, I've pushed commits to the master branch before promoted gem and before developers forked it. But now I wanna show all developers what we do in the codebase and where we follow.
Is the work with a forked project not comfortable for you?

**Comment by gambala** [comment]
Yes, but I understand your motivation too. Ok, let's try to do it canonical way, with forks and PRs
